recipes/google-cloud-kms/meta.yaml

```
@@ -0,0 +1,49 @@
+{% set name = "google-cloud-kms" %}
+{% set version = "1.0.0" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 2167e1c599ea1d9fed96d317ad817d0b37497460c70f11aafa13a24ede7c9c35
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
+
+requirements:
+  host:
+    - enum34;   # [ py<34 ]
+    - google-api-core-grpc >=1.6.0
+    - grpc-google-iam-v1 >=0.11.4
+    - pip
+    - python
+  run:
+    - enum34;   # [ py<34 ]
+    - google-api-core-grpc >=1.6.0
+    - grpc-google-iam-v1 >=0.11.4
+    - python
+
+test:
+  imports:
+    - google
+    - google.cloud
+    - google.cloud.kms_v1
+    - google.cloud.kms_v1.gapic
+    - google.cloud.kms_v1.gapic.transports
+    - google.cloud.kms_v1.proto
+
+about:
+  home: https://github.com/GoogleCloudPlatform/google-cloud-python
+  license: Apache Software
```

that should be just ``Apache``

---

To follow the SPDX naming scheme it should be:
```suggestion
  license: Apache-2.0
```